packages/auth0-nuxt/src/runtime/server/composables/use-auth0.server.spec.ts

```diff
@@ -101,4 +111,34 @@ describe('useAuth0 server composable (server environment)', () => {
 
     expect(() => useAuth0(mockEvent)).toThrow('The `useAuth0` composable should only be used on the server.');
   });
+
+  it('should correctly configure the redirect_uri when using a sub directory with appBaseUrl', () => {
+    mockEvent.context.auth0ClientOptions.appBaseUrl = 'http://localhost:3000/subdir';
+
+    useAuth0(mockEvent);
+
+    expect(ServerClient).toHaveBeenCalledTimes(1);
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const options = (ServerClient as Mock<any>).mock.calls[0]![0] as ServerClientOptions;
+
+    expect(options.authorizationParams!.redirect_uri).toBe('http://localhost:3000/subdir/auth/callback');
```

Comment:
Before this change, the `redirect_uri` was 'http://localhost:3000/auth/callback' in this case